CAAppKitBridge/Source/NSView+CAMethods.m

```diff
@@ -0,0 +1,201 @@
+/* Source/NSView+CAmethods.m
+
+   Copyright (C) 2018 Free Software Foundation, Inc.
+
+   Author: Stjepan Brkic <stjepanbrkicc@gmail.com>
+   Date: June 2018
+
+   This file is part of QuartzCore.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; see the file COPYING.LIB.
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110-1301, USA.
+*/
+
+#import "GSCAData.h"
+#import "NSView+CAMethods.h"
+#import <Foundation/Foundation.h>
+
+@implementation NSView (NSViewCAmethods)
+- (BOOL) wantsLayer {
+  if (self->_coreAnimationData == nil) {
```

Comment:
GNU (and GNUstep) style is not 

```objective-c
  if (self->_coreAnimationData == nil) {
      return NO;
  }
```

Reminder, for all but method definitions:

- Curly on new line
- Curly indented by two spaces
- Codeblock inside curly on new line
- Codeblock inside curly indented by two additional spaces
- Closing curly aligned with opening curly

Curly for methods still goes on a newline, it's just not indented itself.

```objective-c
- (BOOL) wantsLayer
{
  if (self->_coreAnimationData == nil)
    {
      return NO;
    }
  GSCAData....
```